### DISCUSSION TITLE
希望能有前端导入的功能

### DISCUSSION THREAD

**Discussion by voidin** [dream-num/univer#1632] [Q&A | 常见问题]
希望能有前端导入的功能

**Comment by jikkai** [comment]
目前导入导出需要服务器支持，对于无服务依赖的前端导入导出功能目前还在做一些非功能开发上的研判，请耐心等待

**Comment by msy031** [comment]
同样需要 导入/导出功能

**Comment by jikkai** [comment]
目前开放的插件接口足以实现导入导出功能，即使是 univer 团队也只是基于这些插件暴露的接口之上进行导入导出功能的开发。如果社区对此需求非常着急的话可以尝试自行研发。

当然我们了解其中的复杂程度，因此也欢迎为此尝试的同学多和我们交流，我们会提供必要的技术支持。🙌

**Comment by yict** [comment]
这个导入转换的代码希望可以开发一下
